assembl/static2/js/app/components/common/fileUploader.jsx

```diff
@@ -65,6 +65,9 @@ class FileUploader extends React.Component {
   }
 
   render() {
+    const fileSrc = this.state.fileSrc;
+    const fileIsImage = this.props.isImage;
+    const urlIsImage = fileSrc && fileSrc.includes('png' || 'jpeg' || 'jpg' || 'gif');
```

Comment:
This is not bad, but there are other file types that can be images. You can make a better estimation of the type by checking the `File` object's `type`. It will often be `application/<type>`. 

https://developer.mozilla.org/en-US/docs/Web/API/File 
If there is a failing test on this, we can work on it together.

Reply:
answered this issue in my answer to Cédric, I will take care of this by updating the fileUploader test